### PR TITLE
Room 76-Investigate Performance Of Canvas Rendering

### PR DESCRIPTION
**Pull request by bryansamuel1**

### CODE REVIEW

**Comment by NikoGrano** [comment]
Gimme sec...

**Comment by NikoGrano** [comment]
Some stypid Github automatic thing caused this PR close.... Dunno... Like said before, LGTM to merge.

File: src/components/TheCanvas.vue [hunk 1]
```diff
@@ -40,17 +40,25 @@ export default class TheCanvas extends Vue {
     @Getter(`tools/${ToolGetters.TOOLS}`) tools!: Tool[]
     @Sockets.Getter(SocketGetters.SOCKET) socket!: WebSocket
     @Sockets.Action(SocketActions.SEND_IF_OPEN) send!: (message: string) => void
-    @Action(`canvas/${CanvasAction.ADD_CANVAS_ELEMENT}`) addCanvasElement!: (canvasElementPayload: CanvasElementPayload) => void
+    @Action(`canvas/${CanvasAction.ADD_CANVAS_ELEMENT}`) addCanvasElement!: (canvasElement: CanvasElement) => void
```
[the remark this answers is not in the document]
Comment (Ataww) [reply src/components/TheCanvas.vue]: Purely optional tho

[the remark this answers is not in the document]
Comment (bryansamuel1) [reply src/components/TheCanvas.vue]: Yeah sounds good. I'll have a look!

**Pull request merged**